tests/pfcwd/test_pfc_config.py

```diff
@@ -158,11 +158,15 @@ def stop_pfcwd(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     Returns:
         None
     """
-    yield
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     logger.info("--- Stop Pfcwd --")
     duthost.command("pfcwd stop")
 
+    yield
+
+    logger.info("--- Start Pfcwd--")
+    duthost.command("pfcwd start_default")
```

Comment:
Hi @ZhaohuiS, I've seen your concern here for this https://github.com/sonic-net/sonic-mgmt/pull/5392#discussion_r884463037

The problem with putting `yield` first is that it doesn't reset pfcwd to the default setting as we should be doing after this module

Reply:
This fixture seems identical to the one in tests/pfcwd/test_pfcwd_all_port_storm.py
1. can we combine these two and put it into helper file?
2. I see this one is scoped at "function" level and other at "class" level. What would be the reason we have two different scopes?
3. They both have autouse=True, which will then called for other tests automatically. Do we want to call this fixture for other tests as well?

Reply:
maybe in pfcwd_helper.py

Reply:
The scope "function" and "class" were originally there and it passes our nightly so i didn't spend time looking into it. The reason why for `all_port_storm` we have class is that we only have 1 test in 1 class.

In this module we want to use before every tests but for the other module we don't apply the same